### DISCUSSION TITLE
Classify clusters with similar characteristics by default across multiple files?

### DISCUSSION THREAD

**Discussion by Mansi-Purohit** [DrCoffey/DeepSqueak#156] [Q&A] [answered]
Hi,

I'm wondering if it's possible to get clusters with the same characteristics to be named/numbered the same across several files that are analyzed individually?

Thank you,
Mansi

**Comment by DrCoffey** [comment, marked as answer]
Hey @Mansi-Purohit,

You can save an unsupervised clustering model (K-means) and reapply it to any subsequently analyzed files. If you think any clusters appear to have the same type of call you can name all those clusters the same thing within the clustering GUI. So clusters 1, 6, and 9 could all be renamed "chevron" or whatever. Then when you save the model and apply it to future files, they will all get named the same.

-Kevin
